CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Removed
 
 ### Fixed
+* Complies with bug introduces from https://github.com/golang/go/issues/19767
```

Comment:
`Complies with ssh behavior change in ...` maybe?

Reply:
Hmm. Yeah I agree. Will change

Reply:
Fixed here 132aa9b

Reply:
Err b7c3acd